src/components/MDX/Sandpack/createFileMap.ts

```diff
@@ -17,7 +17,7 @@ export const createFileMap = (codeSnippets: any) => {
 
       if (props.meta) {
         const [name, ...params] = props.meta.split(' ');
-        filePath = '/' + name;
+        filePath = '/src/' + name;
```

Comment:
I moved all files to `src`. I could have done on the markdown files and make it more transparent, but this would mean updating all pages (too much work)

Reply:
Can you add a comment indicating this so we can clean up later?